config/limit.ini

```
@@ -34,12 +34,12 @@ none=2
 ; max_relaying=100
 
 
-[recipients_history]
+; [recipients_history]
```

```suggestion
[recipients_history]
```

To avoid the possibility of accidental "fall through" of config settings, shouldn't the section headings *ALWAYS* be uncommented?